tjmonopix2/scans/eudaq_producer.py

```diff
@@ -0,0 +1,490 @@
+#! /usr/bin/env python
+# load binary lib/pyeudaq.so
+import numpy as np
+# import tables as tb
+from tqdm import tqdm
+import yaml
+import threading
+import signal
+import os
+import time
+import argparse
+
+import pyeudaq
+
+#from tjmonopix2.system import logger
+from tjmonopix2.scans import scan_ext_trigger
+from tjmonopix2.analysis import analysis_utils as au
+
+PROJECT_FOLDER = os.path.join(os.path.dirname(__file__), '..')
+BDAQ_DEFAULT_CONF_FILE = os.path.join(PROJECT_FOLDER, 'system', 'bdaq53.yaml')
+TESTBENCH_DEFAULT_FILE = os.path.join(PROJECT_FOLDER, 'testbench.yaml')
+
+scan_configuration = {
+    'start_column': 0,
+    'stop_column': 512,
+    'start_row': 0,
+    'stop_row': 512,
+
+    'scan_timeout': False,
+    # timeout for scan after which the scan will be stopped, in seconds; if False no limit on scan time
+    'max_triggers': False,
+    # number of maximum received triggers after stopping readout, if False no limit on received trigger
+
+    'trigger_latency': 100,  # latency of trigger in units of 25 ns (BCs)
+    'trigger_delay': 57,  # trigger delay in units of 25 ns (BCs)
+    'trigger_length': 32,  # length of trigger command (amount of consecutive BCs are read out)
```

Comment:
None of these are needed in an untriggered device and anyway not used / commented out.

Reply:
You are right, i will clean this (and also the many comments regarding this)